moveit_ros/moveit_servo/test/pose_tracking_test.cpp

```diff
@@ -63,24 +63,48 @@ class PoseTrackingFixture : public ::testing::Test
   void SetUp() override
   {
     // Wait for several key topics / parameters
-    ros::topic::waitForMessage<sensor_msgs::JointState>("/joint_states");
-    while (!nh_.hasParam("/robot_description") && ros::ok())
+    
+    //TODO: Need to figure out replacement. How to use WaitSet?
```

Comment:
No need to do it this way anymore. Simply call [`PlanningSceneMonitor::waitForCurrentRobotState()`](http://docs.ros.org/en/noetic/api/moveit_ros_planning/html/classplanning__scene__monitor_1_1PlanningSceneMonitor.html#a75135c3e95ce2c6dd9a5c379ca4bfdb5) after initializing the PSM.